src/actions/keyBasedAssetTransferActions.js

```diff
@@ -239,19 +242,70 @@ export const calculateKeyBasedAssetsToTransferTransactionGasAction = () => {
     const { history: { gasInfo } } = getState();
     const gasPrice = getGasPriceWei(gasInfo);
 
-    const keyBasedAssetsToTransferUpdated = await Promise.all(
+    let keyBasedAssetsToTransferUpdated = await Promise.all(
```

Comment:
Just wondering what happens if one of these fails?

Reply:
It shouldn't, apart from basic mapping the only call we make is `calculateGasEstimate` and if it fails it returns default gas limit of 500k.

Reply:
Thanks for the explanation 👍